src/domains/transaction/components/MultiSignatureRegistrationForm/__snapshots__/MultiSignatureRegistrationForm.test.tsx.snap

```diff
@@ -412,7 +412,7 @@ exports[`MultiSignature Registration Form should render form step 1`] = `
                         data-testid="InputCurrency"
                         name="fee"
                         type="text"
-                        value=""
+                        value="0"
```

Comment:
I'm checking why this changed

Reply:
@alesmit `minParticipants` seems undefined here but it should be ok if the validation is failing in the input and preventing the user to continue for 0 participants.

Reply:
It seems that the old implementation was replacing 0 with empty string, and removing `|| 0` here brought this snapshot as it was before https://github.com/ArkEcosystem/desktop-wallet/blob/5afaeb2c8a9325479fc55edd73e15e36740a0ac6/src/domains/transaction/components/MultiSignatureRegistrationForm/FormStep.tsx#L102

Reply:
Ah right, indeed. Got confused from the snapshots and thought it was changed in the `minParticipants` input. What you say is correct. Great :+1: